README.md

```diff
@@ -101,60 +101,70 @@ add `priority` label when you start backing us :smile:
 
 See: [docs/deployment.md](docs/deployment.md)
 
-# Development
+## Development
 
-> This abbreviated guide is meant to give a guide as to how github apps are made and basic steps to reproduce the OpenCollective Bot should you wish to make futher changes.
+> This is a guide on how to run the OpenCollective Bot locally.
 
 Visit the [Github App Docs](https://developer.github.com/apps/quickstart-guides/setting-up-your-development-environment/) for a full documentation on Github Apps
 
-## Installation
+### **Local Installation**
 
-Feel free to clone or download this project using the **git clone** and install using **Yarn install** to install required dependencies
+Clone this project using the `git clone` command and install dependencies with `yarn` using the `yarn install` command to install all required dependencies.
 
-**WebHooks** are crucial to production **Github Apps**. Hence, you can use [Ngrok](https://ngrok.com) , [Smee](https://smee.io) or other comfortable options .
-
-> **Smee** which is supported by **Probot** is however used in this guide .
+**WebHooks** are crucial to production **Github Apps**. This project which was built using **Probot** relies on **Smee** for Webhooks
 
 ### Setting up Smee
 
 > Visit the [Smee](https://smee.io) website, to start a new channel
 
--After creating a channel, take note of the **Webhook Proxy URL** shown as it would be used as the **Webhook Url** in the about page when resgistering the app on github
+- The **Webhook Proxy URL** generated by **Smee** should be noted as it would be used as the **Webhook Url** in the **Github App Settings** when registering the app on github.
 
 - `npm i -g smee-client` to install the **smee client**
 
 ### Registering App
 
-- Follow steps listed at [Github](https://developer.github.com/apps/building-github-apps/creating-a-github-app/) to register your bot.
+> **Github App Settings** can be found at `https://github.com/settings/apps`
+
+- For the **Homepage URL**, use the the link to the repository where the Github App is placed. E.g https://github.com/test_user/test_bot
```

Comment:
This doesn't make sense.

Reply:
What **url** would you suggest be placed there ?

I looked on two different docs and they have two different information.